### DISCUSSION TITLE
domino/14.0.0/admin/admn_controllingthesizeoftheadministrationrequestsdatab_t

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#89] [Q&A]
# domino/14.0.0/admin/admn_controllingthesizeoftheadministrationrequestsdatab_t

When administrators make full use of the Administration Process, a large number of request documents and resulting response Log documents are generated in the Administration Requests database (ADMIN4.NSF). This can cause the database to become quite large.

https://help.hcl-software.com/domino/14.0.0/admin/admn_controllingthesizeoftheadministrationrequestsdatab_t.html

**Comment by Davehabz** [comment]
This should include a screenshot of an example of a program document specifically for compacting the admin4.nsf including the optimal switches.  In particular, it does not explain how to release the database so that it can be compacted without shutting down the server.
